src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,69 @@
+.stopwatch {
+  margin: 0;
+  display: flex;
+  position: relative;
+  align-items: center;
+  justify-content: center;
+  height: $stopwatch-height;
+  width: $stopwatch-width;
```

Comment:
Ensure that the variable `$stopwatch-width` is defined in the imported SCSS files, such as `variables.scss`. If it's not defined, the compilation will fail.